tests/sdk.test.ts

```diff
@@ -0,0 +1,286 @@
+import { SayariClient } from "../src/index";
+
+var clientID = String(process.env.CLIENT_ID);
```

Comment:
I don't think you need to call `String()` since env vars are already sting

Reply:
I tried without and I get an error:
```
    tests/sdk.test.ts:13:5 - error TS2322: Type 'string | undefined' is not assignable to type 'Supplier<string>'.
      Type 'undefined' is not assignable to type 'Supplier<string>'.

    13     clientId: clientID,
           ~~~~~~~~

      src/Client.ts:25:9
        25         clientId: core.Supplier<string>;
                   ~~~~~~~~
        The expected type comes from property 'clientId' which is declared here on type 'Options'

```

Reply:
You have to take care of undefined case. Either `const clientId = process.env.CLIENT_ID || ''` or deal with it differently maybe throw an Error?